index.js

```diff
@@ -97,6 +97,11 @@ class PQueue {
 			}
 		});
 	}
+
+	clear() {
+		this.queue._queue = [];
```

Comment:
Actually, this is not entirely correct. It changes a private property of the Queue. For instance, what if I provide a custom queue which uses `_q` instead?
I think it would be better if `opts` was added as private property and in the clear queue method, just create a new one `this.queue = new this._options.queueClass()`.